src/tls.c

```diff
@@ -4777,6 +4771,7 @@ static word16 TLSX_PointFormat_Write(PointFormat* list, byte* output)
 
     return offset;
 }
+#endif /* NO_WOLFSSL_CLIENT*/
```

Comment:
Please explain how to reproduce these unused function warnings. Let's get test covered on it...

Reply:
The problem was manifested when both `NO_WOLFSSL_CLIENT` and `NO_WOLFSSL_SERVER` were defined: In my case, for the stand-alone AES-CTR Arduino example.

I removed the TLSX gating from this PR and will create a separate PR on that topic.